frontend/src/component/ui/Table.scss

```diff
@@ -12,7 +12,7 @@
     border-collapse: collapse;
     width: 100%;
 
-    thead {
+    th {
```

Comment:
There was this weird behaviour when used `thead` as a selector for background color. Changing to `th`
 fixes it.

![thead](https://user-images.githubusercontent.com/18581705/78508054-a9c63280-77a1-11ea-8aef-eddadf530a2e.gif)